deepseq.cabal

```diff
@@ -53,6 +53,11 @@ library
     if impl(ghc < 7.6)
       build-depends: ghc-prim == 0.2.*
 
+  if impl(ghc >= 7.8)
+    other-extensions:
+      ScopedTypeVariables
+      Trustworthy
```

Comment:
@RyanGlScott actually, `other-extensions` is more than just informal. Cabal's solver (since 1.24) takes that list into account when solving install plans and makes sure that the current compiler supports all `other-extensions`.